src/pkcs1v15.rs

```diff
@@ -342,12 +333,13 @@ where
         }
     }
 
-    pub(crate) fn key(&self) -> &RsaPrivateKey {
-        &self.inner
-    }
-
-    pub(crate) fn prefix(&self) -> Vec<u8> {
-        self.prefix.clone()
+    /// Generate a new signing key.
+    pub fn random<R: CryptoRngCore>(rng: &mut R, bit_size: usize) -> Result<Self> {
```

Comment:
Based on discussion in https://github.com/RustCrypto/traits/issues/1148 we're leaning towards:

```suggestion
    pub fn random<R: CryptoRngCore + ?Sized>(rng: &mut R, bit_size: usize) -> Result<Self> {
```